packages/vx-chord/src/Ribbon.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import cx from 'classnames';
+import { ribbon as d3ribbon, Chord, ChordSubgroup } from 'd3-chord';
+
+export type RibbonProps = {
+  chord: Chord;
+  source?: (d: Chord) => ChordSubgroup;
+  target?: (d: Chord) => ChordSubgroup;
+  radius?: (d: ChordSubgroup) => number;
+  startAngle?: (d: ChordSubgroup) => number;
+  endAngle?: (d: ChordSubgroup) => number;
+  children?: ({ path }: { path: string | null }) => string | undefined;
+  className?: string;
+};
+
+export default function Ribbon({
+  chord,
+  source,
+  target,
+  radius,
+  startAngle,
+  endAngle,
+  children,
+  className,
+  ...restProps
+}: RibbonProps & Omit<React.SVGProps<SVGPathElement>, keyof RibbonProps>) {
+  const ribbon = d3ribbon<any, Chord, ChordSubgroup>();
+  if (source) ribbon.source(source);
+  if (target) ribbon.target(target);
+  if (radius) ribbon.radius(radius);
+  if (startAngle) ribbon.startAngle(startAngle);
+  if (endAngle) ribbon.endAngle(endAngle);
+  const path = ribbon(chord) as any;
+  if (children) return children({ path });
```

Comment:
will do a proper review later today but I believe the build is failing because of this line (and a similar one in `Chord.tsx`). I've had issues in other packages where this doesn't seem to satisfy the `React.ReactNode` type, and was able to fix it by wrapping it in fragments ([as suggested in this SO answer](https://stackoverflow.com/a/54908762/1933266)):

```jsx
return <>{children({ path })}</>;
```

That change requires us to bump the `react` peer dep to `^16.3.0` which is a breaking change, but we're moving that direction anyway due to deprecated lifecycle methods.